its/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-ws</artifactId>
-      <version>7.9</version>
```

Comment:
given we test with LATEST, there's no particular reason to stick on 7.9

moving forward, we should consider having a matrix with [LATEST, LTS], because our current ITs run only against LATEST

Reply:
Sounds good. Took a quick look and 8.4.1.35646 is the latest public released version.